### ISSUE TITLE
ResNet50 Model Architecture

### ISSUE THREAD

**Issue by viethungle-vt1401** [IslinaS/ConceptWhitening#1]
- [x] Implement latent blackout function
- [x] Be able to map real pixel -> latent pixel
- [x] Network output: logits + top-k concepts (per image)
